pycbc/derived_parameters.py

```diff
@@ -0,0 +1,462 @@
+# Copyright (C) 2017  Collin Capano
```

Comment:
I have no opinion on module name, so whatever you and @cmbiwer like best, I'm fine to change.